src/adapter/src/coord/command_handler.rs

```diff
@@ -888,13 +888,18 @@ impl Coordinator {
             // Acquire read holds _before_ we determine the least valid read.
             // Otherwise, we're not guaranteed that the since frontier doesn't
             // advance forward from underneath us.
+            let read_holds = self
```

Comment:
This block seems untested.

Reply:
Hmm, that would be surprising. There should be dozens of tests in `materialized_views.slt` and `materialized-view-refresh-options.td` that exercise it. For example, `REFRESH AT CREATION` MVs, and those `REFRESH EVERY` MVs where `ALIGNED TO` is not given.